tests/Hotspot.UnitTests/A.fs

```diff
@@ -0,0 +1,120 @@
+namespace Hotspot
+
+open System
+open System.Collections.Generic
+open Spectre.IO
+open Spectre.IO.Testing
+open Hotspot.Helpers
+
+type FileSystemBuilder() =
+    let env = FakeEnvironment.CreateUnixEnvironment()
+    let fs = FakeFileSystem(env)
+    member this.WithFile (filePath) =
+        fs.CreateFile(FilePath filePath) |> ignore
+        this
+        
+    member this.WithFile(filePath, content) =
+        fs.CreateFile(FilePath filePath).SetTextContent(content) |> ignore
+        this
+        
+    member this.Build() = fs
+
+type FakeCodeRepository(shouldIgnore) =
+    let env = FakeEnvironment.CreateUnixEnvironment()
+    let fileSystem = FakeFileSystem(env)
+    let mutable isGit = false
+    let mutable createdAt = DateTimeOffset.UtcNow
```

Comment:
Maybe use `A.Date.today`

Reply:
Is this worth it inside the actual test helper?

Reply:
Changed all UtcNow to an explicit date.